doc/microsite/src/main/tut/sbt/00_sbt.md

```diff
@@ -353,6 +353,47 @@ You may activate this plugin with the following statement in root project:
 
 ### Transitive Artifact Downloader
 
+Working with dependecies cross projects
+---------------------------------------
+
+Descriptor allows you to search for dependencies and testscopes of your project
+
+Imagine, you have a sdk project and you want to inclide one of it's library artifacts (i.e. target_libraty).
```

Comment:
This should be a part of "Create a BOM for your projects" doc
https://github.com/pshirshov/izumi-r2/pull/425/files#diff-389e35bb33460e4d05c75d0a783571f9R248